packages/store-engine-sqleet/src/index.ts

```diff
@@ -120,31 +102,16 @@ export class SQLeetEngine implements CRUDEngine {
     if (!this.db) {
       throw new Error('SQLite needs to be available');
     }
-    const database: Uint8Array = new Uint8Array(this.db.export());
-    const strings = [];
-    const chunkSize = 0xffff;
-    // There is a maximum stack size. We cannot call `String.fromCharCode` with as many arguments as we want
-    for (let i = 0; i * chunkSize < database.length; i++) {
-      strings[i] = String.fromCharCode.apply(null, <any>database.subarray(i * chunkSize, (i + 1) * chunkSize));
-    }
-    return strings.join('');
-  }
-
-  private async load(database: string): Promise<Uint8Array | undefined> {
-    const databaseBinary = new Uint8Array(database.length);
-    for (let i = 0; i < database.length; i++) {
-      databaseBinary[i] = database.charCodeAt(i);
-    }
-    return databaseBinary;
+    return this.db.export('string');
   }
 
-  async purge(): Promise<void> {
-    // Databases must be closed, when you're finished with them, or the memory consumption will grow forever.
+  public async purge(): Promise<void> {
```

Comment:
```suggestion
  async purge(): Promise<void> {
```

Public is already the default.